core/src/main/java/org/infinispan/container/offheap/BoundedOffHeapDataContainer.java

```diff
@@ -283,32 +281,26 @@ private void ensureSize() {
     *
     * @param entryAddress the new entry address pointer *NOT* the lru node
     */
-   private void addEntryAddressToEnd(long entryAddress, int hashCode) {
-      long nodeAddress = allocator.allocate(OffHeapLruNode.getSize());
+   private void addEntryAddressToEnd(long entryAddress) {
       if (trace) {
-         log.tracef("Creating LRU node 0x%016x for new entry 0x%016x", nodeAddress, entryAddress);
+         log.tracef("Updating LRU contents for new entry 0x%016x", entryAddress);
```

Comment:
Here too, "Adding entry 0x%016x to the end of the LRU list"

Reply:
Sure